graphics_magick-Open_an_images_histogram-20191213.json

```diff
@@ -0,0 +1,15 @@
+{
+  "area": [
+    {
+      "xpos": 1,
+      "ypos": 68,
+      "width": 196,
+      "height": 193,
+      "type": "match"
+    }
+  ],
+  "properties": [],
+  "tags": [
+    "Open_an_image_histogram"

```

Comment:
please use lowercase tags only which is a more common approach. Applies to all needles.

Reply:
ok, changing

Reply:
Done

Reply:
Done